cmd/broker/suite_test.go

```diff
@@ -1029,20 +1030,80 @@ func fixConfig() *Config {
 		Update:         process.StagedManagerConfiguration{MaxStepProcessingTime: time.Minute},
 	}
 }
+func fixBrokerConfigWithOptionalRegion() *Config {
```

Comment:
please, use fixConfig and modify it if possible

Reply:
Fixed in the next commit.